tests/AbstractDoctrineAnnotationFixerTestCase.php

```diff
@@ -25,7 +25,7 @@ abstract class AbstractDoctrineAnnotationFixerTestCase extends AbstractFixerTest
     /**
      * @param array<mixed> $configuration
      *
-     * @dataProvider provideInvalidConfigurationCases
+     * @dataProvider provideConfigureWithInvalidConfigurationCases
```

Comment:
my initial idea was to have a test method `testFooBar` and data provider for it `provideFooBarCases`.
i like this change